operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -618,7 +630,10 @@ type RedpandaConfig struct {
 	AdminAPI       []AdminAPI         `json:"adminApi,omitempty"`
 	PandaproxyAPI  []PandaproxyAPI    `json:"pandaproxyApi,omitempty"`
 	SchemaRegistry *SchemaRegistryAPI `json:"schemaRegistry,omitempty"`
-	DeveloperMode  bool               `json:"developerMode,omitempty"`
+
+	AdditionalSchemaRegistry []SchemaRegistryAPI `json:"additionalSchemaRegistry,omitempty"`
```

Comment:
What's the relationship between this field and `SchemaRegistry`? If it's being added to all specifying multiple schema registry listeners, I'd strongly prefer us to follow the existing convention and add a `SchemaRegistryAPI []SchemaRegistryAPI` field here.

You can then add a helper method (or perform some UnmarshalJSON tricks) to return an aggregated list. It would also be good to rename `SchemaRegistry` to `DeprecatedSchemaRegistry` so it doesn't get referred to. Either way you go, the relationship and desired usage of both these fields should be denoted as comments.